addon/styles/_frost-link.scss

```diff
@@ -121,6 +121,7 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
   &.tertiary {
     background-color: inherit;
     color: $frost-color-blue-1;
+    margin-bottom: -7px;
```

Comment:
Yeah, it was a bit off without the adjustment with the default size.  It is now better, but I did notice that if you increase/decrease the size significantly, it is still a bit off in Firefox.